File: app/src/main/java/com/makingiants/android/banjotuner/BanjenApplication.java
package com.makingiants.android.banjotuner;

import android.app.Application;
import com.crashlytics.android.Crashlytics;
import io.fabric.sdk.android.Fabric;

/**
 * Created by danielgomez22 on 9/15/15.
 */
public class BanjenApplication extends Application {

  @Override
  public void onCreate() {
    super.onCreate();
    Fabric.with(this, new Crashlytics());
  }
}

File: app/src/main/java/com/makingiants/android/banjotuner/BanjenApplication.kt
package com.makingiants.android.banjotuner

import android.app.Application
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric


class BanjenApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Fabric.with(this, Crashlytics())
    }
}
